pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -19,19 +22,15 @@ stdenv.mkDerivation rec {
       --replace 'TOMBEXEC=$0' 'TOMBEXEC=tomb'
   '';
 
-  buildPhase = ''
-    # manually patch the interpreter
-    sed -i -e "1s|.*|#!${zsh}/bin/zsh|g" tomb
-  '';
+  doInstallCheck = true;
+  installCheckPhase = "$out/bin/tomb -h 2>/dev/null";
 
   installPhase = ''
     install -Dm755 tomb       $out/bin/tomb
     install -Dm644 doc/tomb.1 $out/share/man/man1/tomb.1
 
-    ln -s ${gnupg}/bin/gpg $out/bin/gpg
-
     wrapProgram $out/bin/tomb \
-      --prefix PATH : $out/bin:${lib.makeBinPath [ cryptsetup gettext pinentry ]}
+      --prefix PATH : $out/bin:${lib.makeBinPath [ sudo cryptsetup gettext gnupg pinentry utillinux e2fsprogs ]}
```

Comment:
```sudo``` here is not going to work as explained by @Mic92. On NixOS the path to sudo is ```/run/wrappers/bin/sudo``` so you have to tag that on manually instead of using ```makeBinPath```.

Reply:
Come to think if it, ```/run/wrappers/bin``` should already be in everybody's path, so you can just leave that out.

Reply:
I removed `sudo`.